swift/src/Ice/ServantManager.swift

```diff
@@ -1,28 +1,28 @@
 // Copyright (c) ZeroC, Inc.
 
-// TODO: use Mutex and remove @unchecked Sendable
-final class ServantManager: Dispatcher, @unchecked Sendable {
-    private let adapterName: String
-    private let communicator: Communicator
+import Synchronization
 
-    private var servantMapMap = [Identity: [String: Dispatcher]]()
-    private var defaultServantMap = [String: Dispatcher]()
-    private var locatorMap = [String: ServantLocator]()
+final class ServantManager: Dispatcher {
 
-    // This is used to distinguish between ObjectNotExistException and FacetNotExistException
-    // when a servant is not found on a Swift Admin OA.
-    private var adminId: Identity?
+    private struct State {
```

Comment:
Per the Swift Standard Library Programmer's manual "Crucial type aliases and nested types, not exceeding a handful of lines in length" appear first in a type declaration.